books/biology/shorten

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '9.0.0'
```

Comment:
did you want to update this version too?

Reply:
🤔 it's probably not that important, but why not? having fewer versions will make the `install_used_gem_versions` script a little faster so that's good